js/js.tests/test/org/jetbrains/kotlin/js/test/BasicWasmBoxTest.kt

```diff
@@ -152,9 +183,10 @@ abstract class BasicWasmBoxTest(
 
     private fun createPsiFiles(fileNames: List<String>): List<KtFile> = fileNames.map(this::createPsiFile)
```

Comment:
Some common things could be extracted from this class and BasicBoxTest, like createPsiFile and TestFileFactoryImpl